### ISSUE TITLE
Research if the oscillation is already known in literature?

### ISSUE THREAD

**Issue by thinkh** [Caleydo/malevo#62]

**Comment by gfrogat** [comment]
the "oscillation" is mentioned in [Wide Residual Networks
](https://arxiv.org/abs/1605.07146) page 9: according to the authors the oscillation is caused by the weight decay in residual networks

generally it's probably caused by the optimisation process. I'm currently running some tests with Adam to get some more data.

It's possible that it's also caused by the momentum term (good illustration can be found in this Distill post: [Why Momentum Really Works](https://distill.pub/2017/momentum/)).

We might simply end up in so-called "sharp-minima"

I'll create some additional runs without weight decay and different learning rates and batch sizes. The sampling is non exhaustive as I currently only train using a single GTX 1070.

- [ ] Adam
- [ ] SGD, no weight decay
- [ ] SGD, different learning rates
- [ ] SGD, different batch sizes
- [ ] SGD, no momentum

**Comment by gfrogat** [comment]
oscillation is heavily dependent on the model-selection process

we currently specified a set of new experiments in #69 to run

**Comment by NMO13** [comment]
Sepp mentioned momentum term oscillation. E.g. http://slideplayer.com/slide/10997682/

**Comment by gfrogat** [comment]
As we got feedback from Sepp, I'll close this issue.

Please reopen it if there are more questions.
